src/app/filelogger.cpp

```diff
@@ -85,7 +82,7 @@ void FileLogger::changePath(const Path &newPath)
     openLogFile();
 }
 
-void FileLogger::deleteOld(const int age, const FileLogAgeType ageType)
```

Comment:
Why did you change it to use generic integer? I don't see enough reasons for that.

Reply:
>If you think this modification is unnecessary, I'll revert it.

Yes, please.
FileLogAgeType is essentially not generic integer but enumerable type. So it's preferred to be implemented accordingly.